### DISCUSSION TITLE
Repeatable component setup

### DISCUSSION THREAD

**Discussion by djmtype** [formfcw/directus-extension-flexible-editor#48] [Q&A] [answered]
What is the structure for repeatable components, like definition list and gallery?

**Comment by djmtype** [comment]
The definition is just a simple Repeater field. I thought there was more to it than that.

**Comment by formfcw** [comment, marked as answer]
It is up to you! You can do it with separate collections - which I would recommend. Or, as you mentioned, you can use the repeater field.
